IsometricExample.tscn

```diff
@@ -57,7 +58,7 @@ round_precision = 100
 antialiased = true
 
 [node name="Agente" type="Sprite" parent="."]
-position = Vector2( 100, 77 )
+position = Vector2( 101, 76 )
```

Comment:
Sempre que for criar uma cena que não seja uma cena de jogo, zera a position, sempre 0,0.

Reply:
Ok